tests/cache/sdkman.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+setup() {
+  mkdir -p "${HOME}/cache"
+
+  source /dev/stdin <<< "$(cat ./packages/sdkman.sh)"
+  source $HOME/.sdkman/bin/sdkman-init.sh
+  sdk install grails
+}
+
+teardown() {
+  rm -rf $HOME/.sdkman
+}
+
+@test "[sdkman.sh] Caches Grails package" {
```

Comment:
fair point, regarding not needing to test what we should assume `sdkman` is already able to do